include/index.h

```diff
@@ -25,6 +25,8 @@
 #define DEFAULT_MAXC 750
 
 namespace diskann {
+    typedef unsigned label;
```

Comment:
I recommend moving this typedef into `utils.h`: we may want to use it in places where `index.h` is not imported